src/System.IO.Pipes.AccessControl/tests/NamedPipeTests/NamedPipeServerStreamAclTests.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace System.IO.Pipes.Tests
+{
+    public class NamedPipeServerStreamAclTests
+    {
```

Comment:
The PR is a draft because I am currently working in the unit tests. Will submit a commit soon.